app/src/main/java/com/vishnu/whatsappcleaner/ui/HomeScreen.kt

```diff
@@ -91,51 +90,18 @@ fun HomeScreen(navController: NavHostController, viewModel: MainViewModel) {
             val modifier = if (directoryItem.value is ViewState.Success) Modifier
             else Modifier.shimmer()
 
-            Banner(
-                modifier.padding(16.dp),
-                buildAnnotatedString {
-                    when (directoryItem.value) {
-                        is ViewState.Success -> {
-                            var size = (directoryItem.value as ViewState.Success).data.first
-
-                            if (size.contains(" ")) {
-                                val split = size.split(" ")
-                                withStyle(SpanStyle(fontSize = 24.sp)) {
-                                    append(split.get(0))
-                                }
-                                withStyle(SpanStyle(fontSize = 18.sp)) {
-                                    append(" ${split.get(1)}")
-                                }
-                            } else {
-                                withStyle(SpanStyle(fontSize = 24.sp)) {
-                                    append(size)
-                                }
-                            }
-                        }
-
-                        is ViewState.Loading -> withStyle(SpanStyle(fontSize = 18.sp)) {
-                            append("Loading...")
-                        }
-
-                        is ViewState.Error -> withStyle(SpanStyle(fontSize = 18.sp)) {
-                            append("Error")
-                        }
-                    }
-                }
-            )
-
-            Text(
-                modifier = Modifier
-                    .align(Alignment.Start)
-                    .padding(16.dp, 16.dp, 16.dp, 8.dp),
-                text = "Explore",
-                fontWeight = FontWeight.SemiBold,
-                style = MaterialTheme.typography.titleLarge,
-            )
-
             when (directoryItem.value) {
                 is ViewState.Success -> {
-                    LazyColumn(Modifier.weight(1f)) {
+                    LazyColumn(
+                        Modifier.weight(1f),
+                        horizontalAlignment = Alignment.CenterHorizontally
+                    ) {
+                        item {
```

Comment:
can you please add the same to `ViewState.Loading`? the UI shift/jump looks odd once the sizes are loaded.